src/main/java/core/basesyntax/model/FruitsForSale.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class FruitsForSale {
+    private static final List<String> fruitsForSale
+            = List.of("banana", "apple");
```

Comment:
Consider using a `Set` instead of a `List` for `fruitsForSale` if you expect frequent membership checks. A `Set` provides O(1) time complexity for such operations, which is more efficient than a `List`.